### PR TITLE
Adding new documentation file

### PR DESCRIPTION
**Pull request by Gutek**
This is about...
- [x] Issue #8 
- [x] Issue #6

### CODE REVIEW

File: branch.md [hunk 1]
```diff
@@ -0,0 +1 @@
+hello `from` new branch
```
Comment (Gutek) [inline branch.md:1]: this does not seem right...

File: testing.md [hunk 2]
```diff
@@ -0,0 +1 @@
+testing for new...
```
Comment (Gutek) [inline testing.md:1]: this is awesome, thanks, what do you mean by?

**Review by Gutek — COMMENTED** [review]
thank you for the code changes!

File: branch.md (hunk 1 above)
Comment (jolamakota) [inline branch.md:1]: actually i do not like it!

Comment (jolamakota) [reply branch.md:1]: we need to change it

**Review by jolamakota — CHANGES_REQUESTED** [review]
please change it

File: branch.md (hunk 1 above)
Comment (Gutek) [reply branch.md:1]: thanks will do

Comment (jolamakota) [reply branch.md:1]: looks fine thanks

**Review by jolamakota — APPROVED** [review]

**Pull request merged**